nestkernel/per_thread_bool_indicator.h

```diff
@@ -0,0 +1,105 @@
+/*
+ *  per_thread_bool_indicator.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef PER_THREAD_BOOL_INDICATOR_H
+#define PER_THREAD_BOOL_INDICATOR_H
+
+// C++ includes:
+#include <cassert>
+#include <cstddef>
+
+// Includes from nestkernel:
+#include "nest_types.h"
+#include "vp_manager.h"
+
+// Includes from sli:
+#include "dictdatum.h"
+
+namespace nest
+{
+
+class BoolIndicatorUInt64
```

Comment:
Could you add a brief doxygen comment in the class? I think the methods are so straightforward that they don't need further documentation.

Reply:
Done.